notebooks/official/tabular_workflows/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -29,22 +29,22 @@
         "id": "title"
```

Comment:
<p>Can we merge these titles into one?</p><p>"Create a BigQuery destination table." or "Create a BigQuery destination dataset."?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>